docker-compose/docker-compose-esignet/config/esignet-default.properties

```diff
@@ -491,12 +491,12 @@ mosip.esignet.vci.key-values={\
               'credential_issuer': '${mosip.esignet.vci.identifier}',   \
               'credential_endpoint': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', \
               'display': {{'name': 'Insurance', 'locale': 'en'}},\
-              'credentials_supported' : { \
+              'credential_configurations_supported' : { \
```

Comment:
There are code references pointing to credentials_supported, those changes are missing.

Reply:
I'll make changes to parse v11, v12, v13 changes too.

Reply:
The code references to `credentials_supported` are on unit tests where-in the VCI service is mocked. I'll update the tests for v13.